right/src/slave_scheduler.h

```diff
@@ -9,8 +9,8 @@
 // Macros:
 
     #define SLAVE_COUNT (sizeof(Slaves) / sizeof(uhk_slave_t))
-    #define MAX_SLAVE_COUNT 6
-    #define IS_VALID_SLAVE_ID(slaveId) (0 <= slaveId && slaveId <= MAX_SLAVE_COUNT)
```

Comment:
Also, I guess it would be better to throw the MAX_SLAVE_COUNT out so that this sort of error (i.e., forgetting to bump the value when actual SLAVE_COUNT changes) does not repeat in future...

Reply:
Pushed a new commit to cleanup.  If the number of slave is greater than count, a warning is generated